packages/venia-ui/lib/components/CartPage/GiftCards/giftCards.js

```diff
@@ -120,6 +120,8 @@ const GiftCards = props => {
                         id={classes.card}
                         disabled={isApplyingCard || isCheckingBalance}
                         field="card"
+                        mask={value => value && value.trim()}
```

Comment:
Someone on the team has advised me against this; I believe a concern about this function being created on every render. @supernova-at or @revanth0212, is this style of inline function okay? I have a feeling someone would say this should be a returned `talonProp`, but shouldn't be wrapped in `useCallback` since it won't change.

Reply:
Normally I would have returned these from the talon but a mask function is this weird middle ground between logic and UI. Curious what @jimbo thinks.

Personally I'd like to leave it or make it a util function but the savings either way make me feel like it's not super important.